  File "/home/wangxiangwei/Program/DataMaker/data_maker_3.py", line 8
    def __init__(self,image_list_path)#,pose_file_path,motion_output_name):
                                                                          ^
SyntaxError: invalid syntax
  File "/home/wangxiangwei/Program/DataMaker/data_maker_3.py", line 8
    def __init__(self,image_list_path)#,pose_file_path,motion_output_name):
                                                                          ^
SyntaxError: invalid syntax
  File "/home/wangxiangwei/Program/DataMaker/data_maker_3.py", line 8
    def __init__(self,image_list_path)#,pose_file_path,motion_output_name):
                                                                          ^
SyntaxError: invalid syntax
  File "/home/wangxiangwei/Program/DataMaker/data_maker_3.py", line 8
    def __init__(self,image_list_path)#,pose_file_path,motion_output_name):
                                                                          ^
SyntaxError: invalid syntax
  File "/home/wangxiangwei/Program/DataMaker/data_maker_3.py", line 8
    def __init__(self,image_list_path)#,pose_file_path,motion_output_name):
                                                                          ^
SyntaxError: invalid syntax
  File "/home/wangxiangwei/Program/DataMaker/data_maker_3.py", line 8
    def __init__(self,image_list_path)#,pose_file_path,motion_output_name):
                                                                          ^
SyntaxError: invalid syntax
  File "/home/wangxiangwei/Program/DataMaker/data_maker_3.py", line 8
    def __init__(self,image_list_path)#,pose_file_path,motion_output_name):
                                                                          ^
SyntaxError: invalid syntax
  File "/home/wangxiangwei/Program/DataMaker/data_maker_3.py", line 8
    def __init__(self,image_list_path)#,pose_file_path,motion_output_name):
                                                                          ^
SyntaxError: invalid syntax
  File "/home/wangxiangwei/Program/DataMaker/data_maker_3.py", line 8
    def __init__(self,image_list_path)#,pose_file_path,motion_output_name):
                                                                          ^
SyntaxError: invalid syntax
  File "/home/wangxiangwei/Program/DataMaker/data_maker_3.py", line 8
    def __init__(self,image_list_path)#,pose_file_path,motion_output_name):
                                                                          ^
SyntaxError: invalid syntax
  File "/home/wangxiangwei/Program/DataMaker/data_maker_3.py", line 8
    def __init__(self,image_list_path)#,pose_file_path,motion_output_name):
                                                                          ^
SyntaxError: invalid syntax
